openstack/dds/v3/instances/results.go

```diff
@@ -164,3 +171,17 @@ type PortUpdateResp struct {
 	// Database access port.
 	Port int `json:"port"`
 }
+
+type EnabledUpdateResp struct {
+	Enabled bool `json:"enabled"`
+}
```

Comment:
don't need to define this, you can use EnabledOpts

Reply:
done